src/components/App/App.component.jsx

```diff
@@ -41,14 +42,18 @@ function App() {
             <Route exact path="/login">
               <LoginPage />
             </Route>
-            <Private exact path="/secret">
-              <SecretPage />
-            </Private>
+            <DataProvider>
+              <Private exact path="/secret">
```

Comment:
For now is ok but usually, you do not use words like secret or password at URLs for security reasons. Just have that in mind.